ecommerce/frontend/src/components/CategoryBar.less

```diff
@@ -0,0 +1,5 @@
+
+.category-menu {
```

Comment:
I think this css needs to be in HomePage.less to center the CategoryBar 
![image](https://user-images.githubusercontent.com/44057640/99082759-9f8cd200-25d5-11eb-9925-dff478445337.png)

Centered categories look better to me.

Reply:
by centering do you mean vertically or horizontally? because I believe they already are centered horizontally.

Reply:
I meant the CategoryBar to be horizontally aligned. 
![image](https://user-images.githubusercontent.com/44057640/99102682-77aa6800-25ef-11eb-888c-fed44826961a.png)
Your CSS is in the correct position, but copying it to HomePage.less did what I wanted. I was lucky, I guess.

Reply:
Oh okay, must have happened while splitting the LESS code into different files